src/modules.mjs

```diff
@@ -456,16 +456,7 @@ function exportRuntimeSymbols() {
 
   if (PTHREADS && ALLOW_MEMORY_GROWTH) {
     runtimeElements.push(
-      'GROWABLE_HEAP_I8',
-      'GROWABLE_HEAP_U8',
-      'GROWABLE_HEAP_I16',
-      'GROWABLE_HEAP_U16',
-      'GROWABLE_HEAP_I32',
-      'GROWABLE_HEAP_U32',
-      'GROWABLE_HEAP_I64',
-      'GROWABLE_HEAP_U64',
-      'GROWABLE_HEAP_F32',
-      'GROWABLE_HEAP_F64',
+      'GROWABLE_HEAP',
     );
```

Comment:
I suppose technically this is a change in the public API.. so maybe we continue to include these aliases?  e.g. `GROWABLE_HEAP_I8  = () => GROWABLE_HEAP(HEAP8)`?

Reply:
I thought those are just internal helpers?

Reply:
I assume they are .. but `runtimeElements` here (which these are being added to) is a list of things that you can include in `-sEXPORTED_RUNTIME_METHODS`.

Also, internal helpers are still callable from user code.. so I guess it depends if we think its likely anyone is actually calling these?   Maybe no?

Reply:
If they really are internal we can probably remove this whole block here instead of updating it.

Reply:
> Also, internal helpers are still callable from user code.. so I guess it depends if we think its likely anyone is actually calling these? Maybe no?

Yeah that's a good question... I only found discussions like https://github.com/emscripten-core/emscripten/issues/18589 where you suggested those helpers will be gone altogether, which sounds like it means they should be internal - if not already then at least for future-proofing.

They're not documented anywhere, so that seems like a good sign but I'm not sure if there's anything else we can do to make them "truly" internal?

Reply:
I'm up for simply deleting this block then.   Lets check with @kripken before landing though.

Reply:
> I assume they are .. but `runtimeElements` here (which these are being added to) is a list of things that you can include in `-sEXPORTED_RUNTIME_METHODS`.

Ugh that's actually worrying. I was going to do some simplifications to `LE_HEAP_LOAD`/`LE_HEAP_STORE` as well - those are slightly different from those 2 PRs, but there's still some code reduction & size savings to be had there, but those would be breaking changes too if we're not allowed to assume those helpers are internal...

Reply:
I think little endian has very very few users.. if we document the change I think its pretty safe.

Reply:
> I think little endian has very very few users.. 

I have to say, this phrase sounds really funny out of context.